platzky/shortcodes/shortcode.py

```
@@ -0,0 +1,187 @@
+
+@dataclass
+class ShortcodeAttr:
+    """Descriptor for a single shortcode attribute."""
+
+    name: str
+    description: str
+    required: bool = False
+    default: str | None = None
+
+
+class ShortcodeAttrs:
```

Ruff (0.15.12)

[warning] Object does not implement `__hash__` method

(PLW1641)